### PR TITLE
Update Heroku deployment [#169843359]

### PR DESCRIPTION
**Pull request by knowuh**
* add `.env` support for specifying `MONGO_URI` and other env items in development.
* remove `Procfile` because Heroku now looks at `package.json` for building and running.
* Ignore`dist` folder from git. because Heroku runs build script first now.
* Update development documentation.

### CODE REVIEW

**Comment by knowuh** [comment]
@dstrawberrygirl  -- Iwe can now start ignoring the `dist` folder which should save us some eye-bleed when we review pull requests  :)

I think this is ready for review when you get a chance.

File: README.md [hunk 1]
```diff
@@ -1,40 +1,88 @@
...
+A CEASER Game server based on `create-colyseus-app`.
 
 ## :crossed_swords: Usage
-If this is the first time running the app, run `npm install`
+* If this is the first time running the app, run: `npm install`
+* If things are busted, try a nuke-and-repave: `./repave.sh`
+
+## Development:
+
+### Local reqiurements:
```
Comment (kswenson) [inline README.md]:
```suggestion
### Local requirements:  
```

File: README.md [hunk 2]
```diff
@@ -1,40 +1,88 @@
...
+The `.env` file should contain one environmental variable for the mongo server:
 
-`npm start`
+```
+    MONGO_URI=mongodb://[user]:[password]@[host]
+```
 
-Rebuild ready for deploy on Heroku:
+You will need to obtain that URI from another developer or from
+Config Var of the same name in the Heroku [ceaser-server-staging app](https://dashboard.heroku.com/apps/caeser-server-staging/settings).
```
Comment (kswenson) [inline README.md]:
```suggestion
Config Var of the same name in the Heroku [ceaser-server-staging app](https://dashboard.heroku.com/apps/ceaser-server-staging/settings).
```

File: README.md [hunk 3]
```diff
@@ -1,40 +1,88 @@
...
+```
+
+## Heroku Deployment:
+Ceaser-server is Heroku app setup under the Concord Consortium account.
+
+Heroku will look for pushed github branches to run as apps. *`master`* is automatically
+deployed. Other branches can be configured using the Heroku web console as preview apps.
+
+The Heroku pipeline looks at `package.json` to figure out how to build and run
+the Caeser-server app. First `heroku-postbuild` is run which builds the `/dist`
```
Comment (kswenson) [inline README.md]:
```suggestion
the Ceaser-server app. First `heroku-postbuild` is run which builds the `/dist`
```

File: README.md [hunk 4]
```diff
@@ -1,40 +1,88 @@
...
+
+### Preview Branches
+Pushing changes to *any* branch should cause a Heroku preview app to be built.
+You will need to configure the apps settings, to add a `MONGO_URI` config var.
+For now you can just copy the `MONGO_URI` config var from the
+[staging-app settings](https://dashboard.heroku.com/apps/ceasar-server-staging/settings).
+
+### Staging Branch
+Pushing changes to *`master`* will cause the Heroku staging sever to update.
+THe staing server is already provisioned with configured MLAb Mongo Heroku addon.
```
Comment (kswenson) [inline README.md]:
```suggestion
The staging server is already provisioned with configured MLAb Mongo Heroku addon.
```

File: README.md [hunk 5]
```diff
@@ -1,40 +1,88 @@
...
+
+The heroku app websocket server should be available at:
+wss://ceasar-server-staging.concord.org/ or
+
+### Procution Branch
```
Comment (dstrawberrygirl) [inline README.md]: Production? - think you need a new keyboard? ;)

Comment (knowuh) [reply README.md]: Unfortunately I can't blame this one on my keyboard -- the "d" key works and is where its supposed to be.  Its my fingers / brain that are broken.  :)

(All the extra space characters and U characters I can blame on this laptop though ... )

**Pull request merged**